src/main/java/com/google/cloud/security/scanner/pipelines/LiveStateChecker.java

```diff
@@ -115,18 +141,58 @@ LiveStateChecker appendAssertContains(String[] expectedOutputs) {
         liveStates.apply(ParDo.named("Mark states as being live")
             .of(new TagStateWithSource(StateSource.LIVE)));
 
-    // Join the two known-good and the live halves.
     PCollectionView<Map<GCPResource, KV<StateSource, GCPResourceState>>> knownGoodStatesView =
         taggedKnownGoodStates.apply(View.<GCPResource, KV<StateSource, GCPResourceState>>asMap());
+    PCollectionView<Map<GCPResource, KV<StateSource, GCPResourceState>>> liveStatesView =
+        taggedLiveStates.apply(View.<GCPResource, KV<StateSource, GCPResourceState>>asMap());
+
+    // Find outstanding states (known good with no matching live/live with no matching known-good)
+    PCollection<KV<String, GCPResource>> outstandingKnownGoodStates =
```

Comment:
"outstanding" is a bit ambiguous; can we try to find a better naming?
Maybe something along the line of:
`unmatchedKnownGoodStates`
`unmatchedLiveStates`
`FindUnmatchedStates()`
`mergedUnmatchedStates`